arrow-avro/src/writer/mod.rs

```diff
@@ -48,6 +50,7 @@ pub struct WriterBuilder {
     schema: Schema,
     codec: Option<CompressionCodec>,
     capacity: usize,
+    fingerprint_strategy: FingerprintStrategy,
```

Comment:
Because not every format will require a prefix, it maybe better for this to be optional. 

I'm on the fence about this one though since it will add minor complexities to the default `FingerprintStrategy` logic in the `WriterBuilder::build` method. That being said this way does seem more correct.

```suggestion
    fingerprint_strategy: option<FingerprintStrategy>,
```